packages/core/src/org.ts

```diff
@@ -163,10 +163,12 @@ export class ScimSettings extends Serializable {
 }
 
 export class OrgDirectorySettings extends Serializable {
-    scim?: ScimSettings;
     syncProvider: "scim" | "none" = "none";
     syncGroups: boolean = true;
     syncMembers: boolean = true;
+
+    @AsSerializable(ScimSettings)
```

Comment:
@BrunoBernardino This is what was missing I think. My bad, sorry!

Reply:
:facepalm: I was looking for something like this and missed it! Thanks, it worked!